src/components/input/input.js

```diff
@@ -480,6 +530,7 @@ function placeholderDirective($log) {
   }
 }
 
+var visibilityDirectives = ['ngIf', 'ngShow', 'ngHide', 'ngSwitchWhen', 'ngSwitchDefault'];
```

Comment:
@jelbourn I think they might actually be useful: consider having combined/shorter messages on mobile devices to take up less space?

That said, I'm honestly not sure how they would work at the moment. I'll need to do a tad bit of testing.